discord/abc.py

```diff
@@ -679,6 +679,9 @@ async def clone(self, *, name=None, reason=None):
         Clones this channel. This creates a channel with the same properties
         as this channel.
 
+        You must have the :attr:`~Permissions.manage_channels` permission to
```

Comment:
```suggestion
        You must have the :attr:`~discord.Permissions.manage_channels` permission to
```
This is generally how this is done to help with resolving the attribute.

Reply:
Ahh my mistake, I simply used the statement from client.delete_invite to make sure I did it right, but I must have misunderstood.
Thanks you for informing me!